views/index.blade.php

```diff
@@ -110,6 +110,10 @@
           {!! wp_reset_query() !!}
           @if (shortcode_exists('ajax_load_more'))
             {!! do_shortcode('[ajax_load_more container_type="div" css_classes="u-spacing--double" post_type="post" category="'. get_the_category()[0]->slug .'" scroll="false" transition_container="false" button_label="Load More" posts_per_page="10" offset="10"]') !!}
+          @else
+            <div class="c-post-navigation u-space--double--top">
+              <?php posts_nav_link(' ','&laquo; Next Posts','Previous Posts &raquo;'); ?>
```

Comment:
@kelscahill Can you revise this to use the ALPS pagination from here? https://alps.adventist.io/v3/?p=molecules-pagination

Reply:
@designerbrent The pagination style is now updated: cd60a0e